src/mono/wasm/Wasm.Build.Tests/TestAppScenarios/AppTestBase.cs

```diff
@@ -33,63 +34,41 @@ protected void CopyTestAsset(string assetName, string generatedProjectNamePrefix
 
     protected void BuildProject(string configuration)
     {
-        CommandResult result = CreateDotNetCommand().ExecuteWithCapturedOutput("build", $"-bl:{GetBinLogFilePath()}", $"-p:Configuration={configuration}");
+        (CommandResult result, _) = BlazorBuild(new BlazorBuildOptions(Id, configuration));
```

Comment:
Could we also have non-blazor method like that ?

Reply:
We have that.

Reply:
All the building and running go through the same core methods now.